### ISSUE TITLE
Update docs for Dag Storage option 1

### ISSUE THREAD

**Issue by Jonalogy** [airflow-helm/charts#33]
**What is the bug?**

The following two points are related to [Dag Storage Option 1](https://github.com/airflow-helm/charts/tree/main/charts/airflow#option-1---git-sidecar-recommended). As someone new to airflow, helm and kubernetes, these are suggestions to minor issues that set me back a few days.

1. The example shown to create kubernetes secret did not work for me. I had to switch `~/.ssh/*` to `$HOME/.ssh/*`. Suggestion to update the example shell statement to the following:

    ```shell
    kubectl create secret generic \
      airflow-git-keys \
      --from-file=id_rsa=$HOME/.ssh/id_rsa \
      --from-file=id_rsa.pub=$HOME/.ssh/id_rsa.pub \
      --from-file=known_hosts=$HOME/.ssh/known_hosts \
      --namespace airflow
    ```

2. The preceding `ssh://` for url was misleading that caused my git-sync container to throw a `ssh: Could not resolve hostname...` error message. A suggestion to amend the example's presentation:
    ```yaml
    dags:
      git:
        url: git@repo.example.com/example.git # ssh url
        repoHost: repo.example.com
        secret: airflow-git-keys
        privateKeyName: id_rsa

        gitSync:
          enabled: true
          refreshTime: 60
    ```

**What are your Helm values?**

N.A.

**What is your Kubernetes Version?**:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-11T13:09:17Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

**What is your Helm version?**:

```console
$ helm version
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"dirty", GoVersion:"go1.15.4"}
```

**Comment by mdcsaenz** [comment]
You can also use https://username:key@github.com/organization/repo instead of SSH

**Comment by thesuperzapper** [comment]
@Jonalogy thanks for suggesting this, I have a few responses.

1. `~` should work in all user `bash`-like shell sessions, what shell environment are you using?
2. the `ssh://` prefix is allowed in all situations where it would work with `git clone`
   * NOTE: If you are trying to use GitHub, you also have to deal with GitHub's strange syntax of `ssh://git@github.com:torvalds/linux.git` (using a `:` for the organisation, rather than TCP port)

**Comment by Jonalogy** [comment]
> @Jonalogy thanks for suggesting this, I have a few responses.
> 
> 1. `~` should work in all user `bash`-like shell sessions, what shell environment are you using?
> 2. the `ssh://` prefix is allowed in all situations where it would work with `git clone`
>    
>    * NOTE: If you are trying to use GitHub, you also have to deal with GitHub's strange syntax of `ssh://git@github.com:torvalds/linux.git` (using a `:` for the organisation, rather than TCP port)

@thesuperzapper
Apologies for the late reply and thanks for the update!

For point-1, I'm using zsh.

For point-2, my use case was to sync from a private Github repo to my local Minikube cluster and appending `ssh://` threw the `ssh: Could not resolve hostname...` error.

> ```yaml
> git:
>    url: ssh://git@github.com:<username>/<repo_name>.git #<-- This threw the error
>    url: git@github.com:<username>/<repo_name>.git #<-- This works
> ```

**Comment by thesuperzapper** [comment]
@Jonalogy your changes (with some modifications) were implemented already in #39
